syncapi/storage/sqlite3/deltas/2022061412000000_history_visibility_column.go

```diff
@@ -51,6 +54,41 @@ func UpAddHistoryVisibilityColumnCurrentRoomState(ctx context.Context, tx *sql.T
 	if err != nil {
 		return fmt.Errorf("failed to execute upgrade: %w", err)
 	}
+
+	// get the current room history visibility
+	rows, err := tx.Query(`SELECT DISTINCT room_id, headered_event_json FROM syncapi_current_room_state
```

Comment:
Why are you adding this to an _existing_ migration? Won't it not be applied?

Reply:
Fixed.